content/blog/2020-07-31-spreadsheet-workflows-using-r/index.Rmarkdown

```diff
@@ -0,0 +1,185 @@
+---
+title: Spreadsheets Workflows in R
```

Comment:
In this specific phrasing, it feels like this should be "spreadsheet", singular.

Reply:
Agreed - adjusted